.gitignore

```diff
@@ -1,2 +1,6 @@
 /pkg
 /productization
+kickstarts/generated/base.ks
+kickstarts/generated/*.json
+kickstarts/generated/*.cfg
```

Comment:
Good point @carbonin! I had missed it because I have `Gemfile.dev.rb` in my $HOME/.gitignore
:+1: